src/pages/cf-to-k8s/index.mdx

```diff
@@ -16,7 +16,7 @@ The discussion in this module comes in two sub-sections:
 
 The application execution environment diagram for CloudFoundry below demonstrates the components that runs CloudFoundry applications.
 
-![cf](./images/cf.PNG)
+<img src="./images/cf.PNG" alt="cf"></img>
```

Comment:
We need to specify images now with html tag `<img>` ?

Reply:
we may not have to for zoom, but need to double check.